adwaita/upstream/gnome-shell/gnome-shell-sass/widgets/_calendar.scss

```diff
@@ -48,110 +48,78 @@
       @extend %heading;
       padding: 8px 0;
     }
+
     .pager-button {
-      background-color: transparent;
+      @extend .button.flat;
```

Comment:
[Extending Compound Selectors](https://sass-lang.com/documentation/breaking-changes/extend-compound) are not supported by [sass](https://www.npmjs.com/package/sass)(Dart Sass). Need to switch to [node-sass](https://github.com/sass/node-sass)(LibSass)